categories.go

```diff
@@ -72,9 +72,11 @@ func categoriesHandlerWithProxyMode(logger *zap.Logger, indexer Indexer, proxyMo
 			for _, category := range proxiedCategories {
 				if _, ok := categories[category.Id]; !ok {
 					categories[category.Id] = &packages.Category{
-						Id:    category.Id,
-						Title: category.Title,
-						Count: category.Count,
+						Id:          category.Id,
+						Title:       category.Title,
+						Count:       category.Count,
+						ParentId:    category.ParentId,
+						ParentTitle: category.ParentTitle,
```

Comment:
Can we add some test for this?

Reply:
It's not so easy to setup the scenario to test endpoints with the proxyMode.
Created issue for this #1007

Reply:
Added a new file for test `proxy_mode_test.go` for these tests with proxy mode enabled.